### DISCUSSION TITLE
Strip theory - Hydrostatic restoring matrix

### DISCUSSION THREAD

**Discussion by DSkandali** [OpenFAST/openfast#892] [Q&A] [answered]
With the latest version v.3.0.0 and only using the strip theory solution there is no need to define the 6x6 linear hydrostatic restoring matrix and enter this as the additional linear restoring (stiffness) matrix, AddCLin? With the latest version it seems like the hydrostatic restoring stiffness is already calculated from Morison model.

**Comment by jjonkman** [comment, marked as answer]
Dear @DSkandali,

Yes, that is correct.  As of OpenFAST v2.6 and newer, `AddCLin` no longer needs to be specified in HydroDyn to represent the hydrostatic stiffness for strip-theory members or flooded/ballasted members.  This is because hydrostatics in the strip-theory solution are now computed internally within HydroDyn at the displaced position/orientation of the floater at every time step.

Best regards,
